### PR TITLE
Update Ultralytics issue templates

### PR DESCRIPTION
**Pull request by ambitious-octopus**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Refinement of GitHub issue templates to enhance clarity and inclusivity for Ultralytics YOLO.

### 📊 Key Changes
- Updated issue template descriptions from "YOLOv8" to "Ultralytics YOLO".
- Enhanced guidelines for submitting bug reports with more detailed instructions.
- Improved clarity in the question template for providing comprehensive information.
- Made certain fields, such as 'Environment' and 'Minimal Reproducible Example', mandatory for better diagnostic information.

### 🎯 Purpose & Impact
- **Enhanced Clarity**: Clearer instructions help users provide more detailed information, making it easier for the team to diagnose and resolve issues 🛠️.
- **Consistency in Terminology**: Uniform reference to "Ultralytics YOLO" promotes consistency across communications.
- **Better User Support**: More detailed and structured input from users will lead to faster and more efficient support, improving the overall user experience 🚀.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.yml [hunk 1]
```diff
@@ -17,15 +17,15 @@ body:
         Please search the Ultralytics [Docs](https://docs.ultralytics.com), [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) to see if a similar question already exists.
       options:
         - label: >
-            I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.
+            I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.
```
Comment (Burhan-Q) [inline .github/ISSUE_TEMPLATE/question.yml:20]: The hypertext links don't appear to render in the preview of this, but I can't see why they wouldn't.

image

Comment (Burhan-Q) [reply .github/ISSUE_TEMPLATE/question.yml:20]: Current "Question Issue" is rendering them, so perhaps it's just an issue with the preview.

image

**Review by Burhan-Q — APPROVED** [review]
I think it looks good overall, and I appreciate the changes. I definitely agree with making both the MRE and environment fields required, but I have a suspicion that desire will be to keep the required fields to a minimum. That said, I think it would be worth while to test having these required, as I don't know if that has been tested in the past.

I don't think the link rendering is anything to worry about, just pointing it out. Seems like the links didn't have much (if anything) changed, and since they're currently working, expect they should still work.

One minor suggestion for the Question template would be to add a section about Ultralytics Licensing and put a link to the licensing page on ultralytics.com. Hopefully that'll drive people with questions there as GitHub isn't the right place for asking those questions.

**Comment by glenn-jocher** [comment]
@ambitious-octopus @Burhan-Q guys proper English syntax here is "a YOLO question" vs "an Ultralytics YOLO question", with "an" instead of "a".

As @pderrenger mentioned several times before using the https://www.grammarly.com/ plugin will highlight these problems for you.

We must uphold the highest standards.

Screenshot 2024-07-27 at 17 01 02

File: .github/ISSUE_TEMPLATE/question.yml (hunk 1 above)
Comment (ambitious-octopus) [reply .github/ISSUE_TEMPLATE/question.yml:20]: It looks exactly the same, I don't know why it doesn't render!

**Pull request merged**